tasks/giovanni-adapter/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM node:16-alpine
 
 WORKDIR /app
-COPY built package.json package-lock.json ./
+COPY tasks/giovanni-adapter/built tasks/giovanni-adpater/package.json tasks/giovanni-adapter/package-lock.json ./
```

Comment:
This is really confusing that the Dockerfile doesn't work from the directory it is defined in. Instead of the `cd` stuff, can we use symlinks for anything that it needs from the top harmony level?

Reply:
I don't like it either, but no, COPY will not follow symlinks (by design). There are some weird work-arounds using `tar` to pipe to the build command, but they seem even hackier to me.

Reply:
I changed this so that it now runs properly from the directory by doing most of the copying in a script before the Docker build runs